### PR TITLE
update: Easier working with vic

### PR DESCRIPTION
**Pull request by vargac**
* Solves issue #29
* Little janitoring

Signed-off-by: vargac <simon.varga123@gmail.com>

### CODE REVIEW

**Comment by vargac** [comment]
@Adman Check whether it fulfills your requirement.

File: src/vic-stdlib.c [hunk 1]
```diff
@@ -44,6 +46,33 @@ void vic_get(void)
     }
 }
 
+void vic_list(void)
+{
+    char option = '\0';
+    char *args_format = "-%c ";
+
+    if (vic_args(args_format, &option) != 1) {
```
Comment (Adman) [inline src/vic-stdlib.c]: @vargac pure `ls` should work as well. Therefore it needs to have the same functionality like `ls -f`

Comment (mrshu) [inline src/vic-stdlib.c]: @Adman I believe that is questionable as it in the end boils down to what should `ls` actually list.

The other question also is whether we want to have an option for `ls` or rather a whole new command (like ls-var).

Comment (vargac) [inline src/vic-stdlib.c]: @Adman Do you mind if I make two functions: lsv, lsf (or rather mrshu's version suggestion)?

Comment (Adman) [inline src/vic-stdlib.c]: @vargac I think `ls` would list functions and `lsv` variables

File: src/vic-stdlib.c [hunk 2]
```diff
@@ -48,28 +48,19 @@ void vic_get(void)
...
-        }
-    } else if (option == 'v') {
-        for (i = 0; i < vic_vars_len; i++) {
-            vic_println(vic_vars[i].name);
-        }
-    } else {
-        vic_print_err(VIC_ERR_INVALID_ARGS);
-        return;
+    for (i = 0; i < vic_vars_len; i++) {
+        vic_println(vic_vars[i].name);
```
Comment (mrshu) [inline src/vic-stdlib.c]: @vargac the old vic used to also print the address of the function.

I do not think that is necessary anymore.

**Comment by mrshu** [comment]
@vargac @Adman one small comment from me, otherwise it looks good.

@Adman if you do not have any further comments I believe we can merge this.

**Comment by Adman** [comment]
@vargac try to squash your commits into one.

**Comment by mrshu** [comment]
@vargac I very much second @Adman's suggestion

**Comment by vargac** [comment]
@mrshu Squashed, but please see #31 on first

**Pull request merged**